llvm/lib/Transforms/InstCombine/InstCombineCalls.cpp

```diff
@@ -3404,6 +3405,32 @@ Instruction *InstCombinerImpl::visitCallInst(CallInst &CI) {
         MaybeSimplifyHint(OBU.Inputs[0]);
         MaybeSimplifyHint(OBU.Inputs[1]);
       }
+
+      // Try to remove redundant alignment assumptions.
+      if (OBU.getTagName() == "align" && OBU.Inputs.size() == 2) {
```

Comment:
As we emit a bunch of alignment assumptions, can we turn the tag name `align` into a tag ID `OB_align`?